osgi.enroute.hamcrest.wrapper/bnd.bnd

```diff
@@ -8,4 +8,4 @@ Bundle-Copyright: 	Copyright 2012- hamcrest.org
 -buildpath: \
 	org.hamcrest:hamcrest-all;version=1.3
 
--includeresource: LICENSE.txt
+-includeresource: LICENSE.txt, OSGI-OPT/src=@${repo;org.hamcrest:hamcrest-all:jar:sources}
```

Comment:
Did you test this? I don't see that this could work. The repo macro does not know what `org.hamcrest:hamcrest-all:jar:sources` is. It expects a Bundle-SymbolicName. The BSN can be synthesize for a plain jar from groupId:artifactId. But I am not aware it also understands extension and classifier.

Reply:
I added it to the repo with the latest commit (I deleted it previously by mistake). Yes I tested it